contracts/.prettierrc.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+    plugins: ['prettier-plugin-solidity'],
```

Comment:
can you extend from our current prettier-config and add this as an extension

Reply:
Done.